app/controllers/feedbacks_controller.rb

```diff
@@ -37,10 +39,16 @@ def load_support_languages
     @support_languages = Language.where(support_available: true).default_order
   end
 
+  def current_skin
```

Comment:
In `SkinsHelper#skin_tag`, there is also a check for `params[:site_skin]`, e.g. https://test.archiveofourown.org/?site_skin=928.

Because you're essentially replicating the skin setting logic from the `#skin_tag` method, I think it would be good to separate that out into its own method that can be used both for the support tickets and for `#skin_tag`.